CLI.md

```diff
@@ -12,39 +12,44 @@ $ theo <[file]> [options]
 
 ### Options
 
-|Name|Description|Default|
-|----|-----------|-------|
-|`--transform`|valid theo transform|`raw`|
-|`--format`|Comma separated list of valid theo formats|`raw.json`|
-|`--dest`|The path where the result should be written|stdout|
-|`--setup`|The path to an optional JS module that can set up Theo before transformation.||
+| Name                   | Description                                                                   | Default    |
+| ---------------------- | ----------------------------------------------------------------------------- | ---------- |
+| `--transform`          | valid theo transform                                                          | `raw`      |
+| `--format`             | Comma separated list of valid theo formats                                    | `raw.json` |
+| `--dest`               | The path where the result should be written                                   | stdout     |
+| `--setup`              | The path to an optional JS module that can set up Theo before transformation. |            |
+| `--resolveMetaAliases` | Resolve aliases in metadata                                                   | `false`    |
 
 ### transforms / formats
 
 Formats are valid theo supported formats, check the [documentation](https://github.com/salesforce-ux/theo#available-formats) for a full list of supported transforms and formats.
 
 Usage example with formats:
+
 ```
 $ theo tokens.yml --transform web --format scss,cssmodules.css
+$ theo tokens.yml --transform web --format scss,cssmodules.css --resolveMetaAliases
 ```
 
 ### setup module
 
 A valid setup module exports a function that takes theo as the first argument.
 
 Example module (example.js):
+
 ```
 module.exports = theo => {
   theo.registerValueTransform(
-    'addpx', 
-    prop => prop.get('type') === 'size', 
+    'addpx',
```

Comment:
Formatting changes are due to my VSCode prettier settings. Hopefully these changes do not violate any code standard.